web-ui/src/types/connectors.tsx

```diff
@@ -17,7 +17,7 @@ import {
 export enum ConnectorType {
   KAFKA_IN = 'KafkaIn',
   KAFKA_OUT = 'KafkaOut',
-  FILE = 'File',
+  URL = 'HTTP GET',
```

Comment:
why get?

Reply:
it sends GET requests to fetch data .. maybe we should have something that pushes data over http too... we can call that HTTP POST